src/react/components/pages/projectSettings/projectMetrics.tsx

```diff
@@ -42,7 +42,19 @@ export default class ProjectMetrics extends React.Component<IProjectMetricsProps
     public render() {
         if (this.state.loading) {
             console.log("Waiting for data");
-            return <h2> Loading...</h2>;
+            return (
+                <div className="project-settings-page-metrics p-3 bg-lighter-1">
+                    <h3>
+                        <i className="fas fa-chart-bar fa-1x"/>
+                        <span className="px-2">
+                            {strings.projectMetrics.title}
+                        </span>
+                    </h3>
+                    <div className="loading">
+                        <i className="fas fa-spinner fa-6x"/>
```

Comment:
Let's standardize on a single spinner icon.  We are using spinner for the `editorSidebar` component `.fa-circle-notch`